docs/usage.md

```diff
@@ -1,9 +1,10 @@
 Updating
 ---
-In your deployment directory, enter the nix shell with `nix-shell` and run
+In your deployment directory, enter the nix shell with `nix-shell` and run the
+following to update `nix-bitcoin-release.nix`:
 
 ```
-fetch-release > nix-bitcoin-release.nix
+update-nix-bitcoin
 ```
 
```

Comment:
Should we also put a note about deploying, or is that self-explanatory?

Reply:
It think it's self-evident due to the shell output:
```
Fetched and verified release 0.0.50
Updated nix-bitcoin-release.nix
```

Reply:
Ok